### ISSUE TITLE
 Deprecation: The 'pygments'

### ISSUE THREAD

**Issue by brusMX** [liamsymonds/simplygrey-jekyll#4]
```
 Deprecation: The 'pygments' configuration option has been renamed to 'highlighter'. Please update your config file accordingly. The allowed values are 'rouge', 'pygments' or null.
            Source: /Users/user2/site/usersite
       Destination: /Users/user2/site/usersite/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
  Dependency Error: Yikes! It looks like you don't have pygments or one of its dependencies installed. In order to use Jekyll as currently configured, you'll need to install this gem. The full error message from Ruby is: 'cannot load such file -- pygments' If you run into trouble, you can find helpful resources at http://jekyllrb.com/help/! 
  Liquid Exception: pygments in /Users/user2/site/usersite/_posts/2013-12-23-ruby-json.markdown
             ERROR: YOUR SITE COULD NOT BE BUILT:
                    ------------------------------------
                    pygments
```

And then 

```
⇒  gem install pygments
ERROR:  Could not find a valid gem 'pygments' (>= 0) in any repository
ERROR:  Possible alternatives: rygments, pigments, rb-pygments, segments, rugments
```

**Comment by liamsymonds** [comment]
Apologies - I've been offline. I'm planning on getting an environment back up and running for the demo and will include this in the pull request. 

Cheers!

**Comment by liamsymonds** [comment]
Fixed by #5
